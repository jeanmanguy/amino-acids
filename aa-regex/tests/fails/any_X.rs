use aa_regex::*;

fn main() {
    let wrong = any!('x');
}
