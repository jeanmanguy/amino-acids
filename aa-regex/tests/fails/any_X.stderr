error: expected amino acid 1-letter code
 --> $DIR/any_X.rs:4:22
  |
4 |     let wrong = any!('x');
  |                      ^^^
